modules/ebi-metagenomics/dram/distill/main.nf

```diff
@@ -0,0 +1,54 @@
+process DRAM_DISTILL {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/dram:1.3.5--pyhdfd78af_0':
+        'quay.io/biocontainers/dram:1.3.5--pyhdfd78af_0' }"
+
+    containerOptions="--bind ${moduleDir}/tests/fixtures/dram_distill_dbs/:/data/ --bind ${moduleDir}/tests/fixtures/dram_distill_dbs/CONFIG:/usr/local/lib/python3.10/site-packages/mag_annotator/CONFIG"
+
+    input:
+    tuple val(meta), path(dram_input)
+
+    output:
+    tuple val(meta), path("*_dram.html"), emit: distill_out_html // removed "optional" e.g. we don't launch the module when missing input files
+    tuple val(meta), path("*_dram.tsv") , emit: distill_out_tsv
```

Comment:
I think here you don't need the `distill` prefix for each output, as it's a self-contained module that will only output dram output anyway. you can rename these to `html` and `tsv` IMO